cmd/mr_create.go

```diff
@@ -111,6 +112,22 @@ func runMRCreate(cmd *cobra.Command, args []string) {
 	if err != nil {
 		log.Fatal(err)
 	}
+
+	ofilename, err := cmd.Flags().GetString("file-open-editor")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	if ofilename != "" && filename != "" {
+		log.Fatalf("Cannot specify both -F and -o options.")
+	}
+
+	openEditor := false
+	if ofilename != "" && filename == "" {
```

Comment:
If the previous `if` statement already checked if both options weren't used together, you don't need to check `filename == ""` here. If `filename != ""` it'll be caught in the previous conditional.

Reply:
@prarit I think you forgot about this review of mine in your last update.

Reply:
Oh I did!  Fixed in new version.